docs/libraries/azure-core/reference/data-types.md

```diff
@@ -141,6 +141,14 @@ model Azure.Core.Page<TResource>
 | --------- | ------------------ |
 | TResource | The resource type. |
 
+### `PollingOptions` {#Azure.Core.PollingOptions}
+
+Generic polling options for LRO operations.
+
+```typespec
+model Azure.Core.PollingOptions
```

Comment:
Does this not have any properties?

Reply:
This is similar to other models in this md file and elsewhere - template parameters are described, but not model properties.